stdlib/string.mli

```diff
@@ -150,6 +150,28 @@ val mapi : (int -> char -> char) -> string -> string
     string that is returned.
     @since 4.02.0 *)
 
+val fold_left : ('a -> char -> 'a) -> 'a -> string -> 'a
+(** [fold_left f x s] computes
```

Comment:
```suggestion
(** [String.fold_left f x s] computes
```

(and idem for all new functions)

Reply:
Why this suggestion? None of the other docstrings seem to contain the module qualification. The author of the PR had added this as per your suggestion, but I removed it when rebasing because it was confusing when used with the `sync_stdlib_docs` script that synchronizes the docs of the labelled and unlabelled modules.

Reply:
The stdlib docs are inconsistent, sometimes doc strings have it. But in general I think we should move on to remove them.